internal/services/sentinel/sentinel_data_connector.go

```diff
@@ -88,12 +87,8 @@ func assertDataConnectorKind(dc securityinsight.BasicDataConnector, expectKind s
 		kind = securityinsight.DataConnectorKindAmazonWebServicesS3
 	case securityinsight.TiTaxiiDataConnector:
 		kind = securityinsight.DataConnectorKindThreatIntelligenceTaxii
-	case azuresdkhacks.TiTaxiiDataConnector:
-		kind = securityinsight.DataConnectorKindThreatIntelligenceTaxii
 	case securityinsight.TIDataConnector:
 		kind = securityinsight.DataConnectorKindThreatIntelligence
-	case azuresdkhacks.TIDataConnector:
-		kind = securityinsight.DataConnectorKindThreatIntelligence
```

Comment:
how come we are removing these and what effect will it have for the end users?

Reply:
This func `assertDataConnectorKind` is only called by `importSentinelDataConnector` on above lines. 
The `importSentinelDataConnector` func used to use a patched client from `azuresdkhacks`, which will return these two removed models. 
Since the service is consistent with Swagger now, we don't need to use `azuresdkhacks` client now, so we don't need these two models either.